.github/workflows/build_release.yml

```diff
@@ -0,0 +1,73 @@
+
+name: Publish release image to GitHub Container Registry
+
+on:
+  push:
+    tags: [ "v[0-9].[0-9].[0-9]" ]
+
+jobs:
+  build_operator:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build and push operator image
+        uses: docker/build-push-action@v1.1.1
+        with:
+          # Username used to log in to a Docker registry. If not set then no login will occur
+          username: ${{ secrets.PACKAGES_USERNAME }}
+          # Password or personal access token used to log in to a Docker registry. If not set then no login will occur
+          password: ${{ secrets.PACKAGES_PASSWORD }}
+          # Server address of Docker registry. If not set then will default to Docker Hub
+          registry: ghcr.io
+          # Docker repository to tag the image with
+          repository: ${{ secrets.PACKAGES_USERNAME }}/http-add-on-operator
+          # Automatically tags the built image with the git SHA
+          tag_with_ref: true
+          tags: latest
```

Comment:
Would be good to add a tag for the version that was shipped, WDYT?

Reply:
This happens with the "tag_with_ref" key, it adds the tag for the branch or, if the commit is a tag, it adds the tag with the name of the tag automatically :)

Reply:
Ha, didn't know a tag was used here as well; thanks!

#TIL